errbot/backends/slack.py

```diff
@@ -838,6 +838,26 @@ def build_reply(self, mess, text=None, private=False):
             response.to = mess.frm.room if isinstance(mess.frm, RoomOccupant) else mess.frm
         return response
 
+    def add_reaction(self, mess, reaction):
+        return self._react('reactions.add', mess, reaction)
+
+    def remove_reaction(self, mess, reaction):
+        return self._react('reactions.remove', mess, reaction)
+
+    def _react(self, api_call, mess, reaction):
+        result = self.sc.api_call(api_call,
+                                  channel=mess.to.channelid,
+                                  timestamp=mess.extras['ts'],
+                                  name=reaction)
+        if result['ok']:
+            return
+        elif result.get('error') == 'invalid_name':
+            raise ValueError(result['error'], 'No such emoji', reaction)
+        elif result.get('error') in ('no_reaction', 'already_reacted'):
+            raise LookupError(result['error'])
+        else:
+            raise Exception(result['error'])
```

Comment:
Can you raise [SlackAPIResponseError](https://github.com/r3m0t/errbot/blob/64b89797abad251fcd3eee4f7f9526fe71d18e62/errbot/backends/slack.py#L89-L98) here instead?